homeassistant/components/pandora/media_player.py

```diff
@@ -94,14 +94,21 @@ def __init__(self, name: str) -> None:
         self._attr_media_duration = 0
         self._pianobar: pexpect.spawn[str] | None = None
 
-    def turn_on(self) -> None:
+    async def async_turn_on(self) -> None:
         """Turn the media player on."""
         if self.state != MediaPlayerState.OFF:
             return
         self._pianobar = pexpect.spawn("pianobar", encoding="utf-8")
+        # #122027 if pexpect attempts to sleep at any point, a critical error is thrown
+        # "Caught blocking call to sleep ... inside the event loop"
```

Comment:
There is no need to comment the issue # you are fixing within the code. I suggest cleaning this up, or keeping these changes obvious enough to not need comments.